lib/providers/services/http_service.dart

```diff
@@ -108,7 +108,39 @@ class HttpService {
 
       return members;
     } else {
-      print('Cannot fetch members...');
+      print('Cannot fetch members...'+res.body);
+
+      return [];
+    }
+  }
+
+  Future<List<MemberModel>> getMembersWithFilter(String filter) async {
+    print("[http] getMembersWithFilter $filter ");
+    if (filter.length < 2) {
+      return [];
+    } 
+    String lowerBound = filter;
+    String upperBound = filter.padRight(12 - filter.length, "z");
```

Comment:
Why is it extending the range of the search? I would expect it to search with both upper and lower bound being the search string.

Reply:
Yes this is the upper and lower bound - I found that in order to search for what is basically 

abc* 

i.e. "abc followed by any other characters"

We have to set lower bound of "abc" (just the 3 letters as this could be a valid EOS name for someone who purchased the domain "abc")

And upper bound of "abczzzzzzzzz" (the highest possible account starting with abc

So really this is searching for` filter*`